src/components/layers/layers/Layer.js

```diff
@@ -45,8 +46,8 @@ const styles = theme => ({
 
 const Layer = ({ classes, layer, onClick }) => {
     const { img, type, name } = layer;
+    const label = name || i18n.t(type);
```

Comment:
Does i18n.t(type) actually translate correctly? I was pretty sure you had to insert an actual string.